lib/custom_widgets/alert_dialog.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:flutter/material.dart';
+import 'package:sossoldi/main.dart';
```

Comment:
Use relative path to import internal files, as per #43

Reply:
Ok done with the latest commits